### PR TITLE
OCPBUGS#5449: Change the cluster MTU

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-5449

Link to docs preview: https://58240--docspreview.netlify.app/openshift-enterprise/latest/networking/changing-cluster-network-mtu.html#nw-cluster-mtu-change_changing-cluster-network-mtu

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kcarmichael08** [comment]
Looked at the comment trail for the PRs and it looks good to me.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by jboxman-rh** [comment]
@jeana-redhat, this looks like it's good to go for all cherry-picks? Thanks!

File: modules/nw-cluster-mtu-change.adoc [hunk 1]
```diff
@@ -91,165 +91,61 @@ where:
...
 +
 --
-**** Set the following values:
-***** `802-3-ethernet.mtu`: Specify the MTU for the primary network interface of the system.
-***** `connection.interface-name`: Optional: Specify the network interface name that this configuration applies to.
-***** `connection.autoconnect-priority`: Optional: Consider specifying an integer priority value above `0` to ensure this profile is used over other profiles for the same interface. If you are using the OVN-Kubernetes network plugin, this value must be less than `100`.
-**** Remove the `connection.uuid` field.
-**** Change the following values:
-***** `connection.id`: Optional: Specify a different NetworkManager connection profile name.
+where:
```
Comment (sheriff-rh) [inline modules/nw-cluster-mtu-change.adoc:105]: This is a perfect opportunity for [callouts](https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/) in the future.

**Review by sheriff-rh — COMMENTED** [review]
lgtm - I made one comment for next time, minor suggestion :) 

Merging!

**Pull request merged**